backend/src/helpers/students.js

```diff
@@ -1,4 +1,25 @@
 const db = require('../models/student')
+// const debug = require('debug')('app:students')
+
+exports.getAllStudents = async function () {
+  try {
+    const students = await db.Student.query().select()
+    return students
+  } catch (err) {
+    return { err }
+  }
+}
+
+exports.getStudentById = async function (id) {
+  try {
+    const student = await db.Student.query()
+      .select()
+      .where('StudentID', id)
```

Comment:
Does `await db.Student.query().findById(id)` work? I know we changed the id column to `StudentID`, but I would have hoped that Objection handled that for us and it still works.

Otherwise, you can try `await db.Student.query().findOne({ StudentID: id })`, so that at least we return the student object (instead of an array with one student inside it).

Reply:
`await db.Student.query().findById(id)` & `await db.Student.query().findOne({ StudentID: id })` both worked. I went for the first one and worked in some error handling. Thank you for the improvement!